docs/docker/development.md

```diff
@@ -1,7 +1,8 @@
 # Using Docker Images During Development
 
 We've created a special `docker-compose.dev.yml` override file that should configure docker images to be easier to use
-during development.
+during development.  This file, alone with several others are generated by a python script `generate_docker_quickstart.py`, 
```

Comment:
Thanks for the contribution Danny!

This specific file is not auto-generated, however. The ones that are generated are only used when running default `datahub docker quickstart`, that is: docker/quickstart/docker-compose.quickstart.*.yml